File: Daily Challanges/2026-03-March/2026-03-17.js
function getMilestone(years) {
  if (years < 1) {
    return "Newlyweds";
  } else if (years < 5) {
    return "Paper";
  } else if (years < 10) {
    return "Wood";
  } else if (years < 25) {
    return "Tin";
  } else if (years < 40) {
    return "Silver";
  } else if (years < 50) {
    return "Ruby";
  } else if (years < 60) {
    return "Gold";
  } else if (years < 70) {
    return "Diamond";
  } else {
    return "Platinum";
  }
}

File: Daily Challanges/2026-03-March/2026-03-17.py
def get_milestone(years):
    if years < 1:
        return "Newlyweds"
    elif years < 5:
        return "Paper"
    elif years < 10:
        return "Wood"
    elif years < 25:
        return "Tin"
    elif years < 40:
        return "Silver"
    elif years < 50:
        return "Ruby"
    elif years < 60:
        return "Gold"
    elif years < 70:
        return "Diamond"
    else:
        return "Platinum"